zbarcam/zbarcam-qt.cpp

```diff
@@ -513,6 +520,23 @@ public Q_SLOTS:
 		    &SettingsButton::button_clicked);
 	}
 
+        // ean-search.org API lookup
+        QLabel * token_label = new QLabel(tr("ean-search.org API token:"));
```

Comment:
I don't think it's a good idea to require users to create/register API tokens to use a simple feature like this.

Reply:
All lookup APIs that I know need some form of authentication.

Reply:
http://www.brocade.io/documentation doesn't seem to require an API key

Reply:
I tried brocade.io but it seems they don't find many products. It would be best to add all information providers to choose from.